Accelerometer/CS/README.md

```diff
@@ -0,0 +1,286 @@
+
+The ADXL345 breakout board has 8 IO pins, connect them as follows:
+
+1. **GND:**  Connect to ground on the UP Board (Pin 6)
+2. **VCC:**  Connect to 3.3V on the UP Board (Pin 1)
+3. **CS:**   Connect to SPI0 CS0 on the UP Board (Pin 24). This is the chip-select line for the SPI bus.
+4. **INT1:** Leave unconnected, we're not using this pin
+5. **INT2:** Leave unconnected, we're not using this pin
+6. **SDO:**  Connect to SPI0 MISO on the UP Board (Pin 21)
+7. **SDA:**  Connect to SPI0 MOSI on the UP Board (Pin 19)
+8. **SCL:**  Connect to SPI0 SCLK on the UP Board (Pin 23). This is the clock line for the SPI bus.
+
+Here are the connections shown on a breadboard:
+
+![Breadboard connections](../../Resources/breadboard_assembled_UP Board1.png)
+
+Here are the schematics:
+
+![Accelerometer schematics](../../Resources/schematics_UP Board.png)
+
+### Deploy the app 
+
+ Click the Start button to search for the app by name, and then launch it.
+
+When everything is set up, power your device back on, and open up the sample app in Visual Studio. Open the file **MainPage.xaml.cs** and change the following line from **Protocol.NONE** to **Protocol.SPI**:
+
+```csharp
+public sealed partial class MainPage : Page
+{
+    /* Important! Change this to either Protocol.I2C or Protocol.SPI based on how your accelerometer is wired   */
+    private Protocol HW_PROTOCOL = Protocol.SPI; 
+    // ...
+}
+``` 
+
+### [Generate an app package](https://docs.microsoft.com/windows/msix/package/packaging-uwp-apps#generate-an-app-package)
+
+### [Install your app package using an install script](https://docs.microsoft.com/windows/msix/package/packaging-uwp-apps#install-your-app-package-using-an-install-script)
+
+### BIOS Settings for UP Board
+If you are using UP Board, you have to setup the BIOS SPI configuration.
+
+1. Once you power on the UP board, select the **Del** or **F7** key on your keyboard to enter the BIOS setting.
+
+1. Navigate to **Boot** > **OS Image ID** tab, and select **Windows 10 IoT Core**.
+
+1. Navigate to the **Advance** tab and select the **Hat Configuration** and select "LPSS SPISupport" as "Enabled".
+
+1. For more information, please review the [UP Board Firmware Settings](https://www.annabooks.com/Articles/Articles_IoT10/Windows-10-IoT-UP-Board-BIOS-RHPROXY-Rev1.3.pdf).
+  
+### Run the app
+The SPIAccelerometer app will deploy and start, and you should see accelerometer data show up on screen.
+If you have your accelerometer flat on a surface, the Z axis should read close to 1.000G, while X and Y are close to 0.000G. The values will fluctuate a little even if the device is standing still.
+This is normal and is due to minute vibrations and electrical noise. If you tilt or shake the sensor, you should see the values change in response. Note that this sample configures the device in 4G mode, so you wont be able to see G readings higher than 4Gs.
+
+![SPI Accelerometer running](../../Resources/spiaccelerometer_screenshot.png)

```

Comment:
Make sure this image appears

Reply:
Updated the images path, Its coming up now.